source/CommonJobs/CommonJobs.Application.EvalForm/EmployeeSearching/EvaluationToDelete_Search.cs

```diff
@@ -0,0 +1,40 @@
+﻿using CommonJobs.Domain.Evaluations;
+using Raven.Client.Indexes;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CommonJobs.Application.EvalForm.EmployeeSearching
+{
+    class EvaluationToDelete_Search : AbstractMultiMapIndexCreationTask<EvaluationToDelete_Search.Projection>
+    {
+        public class Projection
```

Comment:
In that case, you should try to create _projection type_ objects as result of map and reduce operations. Anyway it does not seems to be a RavenDB common practice.

Anyway, I was not talking about projection itself, I was talking about the index type (`AbstractMultiMapIndexCreationTask`), there should be a simpler index base class that fits for your needs.